examples/Python/python-django-webapp-sample/README.md

```diff
@@ -68,6 +69,19 @@ python manage.py migrate
 python manage.py runserver
 ```
 
+## Refresh Configuration
+
+To refresh your configuration, you first update the value in Azure App Configuration, then update the Sentinel value to trigger a refresh.
+
+```Powershell
+az appconfig kv set --name <config-store-name> --key testapp_settings_color --value "violet"
+az appconfig kv set --name <config-store-name> --key sentinel --value "2"
+```
+
+Refresh the page in your browser to see the updated value.
+
+NOTE: By default refresh can only be triggered every 30 seconds. It may have to wait up till 30 seconds and then refreshing a page to see a refresh.
```

Comment:
Should this be `You may have to wait up to 30 seconds before refreshing the page to see the changes.`?

Reply:
The thing I want to make clear here is that you have to wait 30 seconds, then refresh the page. Just waiting 30 will do nothing.